src/v2/guide/accessibility.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Accessibility
+type: guide
+---
+
+# Accessibility
+
+An accessible website is a website that can be used be anyone—be that a person with a disability, someone on a slow connection, or someone whose mouse has ran out of battery. It's easy to make a website assuming that all our users are using a keyboard, mouse, and screen, and have a way of hearing the sound produced by our websites, but that often isn't true: millions of people around the world have disabilities and are unable to use all the functionality of a computer in the same way most developers do. While they often have tooling to help them, they're relying on the people building the websites to make them accessible and work well with the tooling.
```

Comment:
_..whose mouse has ran out of battery._

Should be `run` not `ran`.

Reply:
Also:
_... a website that can be used be anyone ..._ => _... used **by** anyone..._